src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -1172,7 +1177,7 @@ export function getCustomizedConfiguration(chartOptions: IChartOptions, chartCon
     ];
 
     const commonData = configurators.reduce((config: any, configurator: any) => {
-        return merge(config, configurator(chartOptions, config, chartConfig));
+        return merge(config, configurator(chartOptions, config, chartConfig, drillConfig));
```

Comment:
Please add simple test for this

Reply:
I suppose you want to add test cases to assert `drillConfig` exists in `commonData`, right?

Reply:
Fixed :)